src/content/Colors/Colors.tsx

```diff
@@ -1,35 +1,90 @@
 import React from 'react';
 import styled from 'styled-components';
-import { colors, TTextHexColors, IColorSpec } from './../../constants/colors';
+import {
+  colors,
+  IColors,
+  IBrandColors,
+  IActionColors,
+  INeutralColors,
+  INotificationColors,
+} from './../../constants/colors';
 
-export type TColorNames = keyof typeof colors.base | keyof typeof colors.neutral | keyof typeof colors.semantic;
+type TColorVariants = keyof IColors;
```

Comment:
Hi Lluis! Great Work!

Knowing that now we are gonna have a plain Colors object, what do you think if we remove all the other related interfaces (like IBrandColors, IActionColors, etc)?

In the end, we only need to export the IColors interface.